elements_imaging/imaging.py

```diff
@@ -421,13 +464,18 @@ def make(self, key):
                 self.Block.insert(nonrigid_blocks)
 
             # -- summary images --
-            field_keys = (scan.ScanInfo.Field & key).fetch('KEY', order_by='field_z')
+            field_keys = {}
+            for field_key in (Processing.Field * Processing.MemberField
+                              * scan.ScanInfo.Field & key).fetch('KEY', order_by='field_z'):
+                if field_key['processing_field_idx'] not in field_keys:
+                    field_keys[field_key['processing_field_idx']] = field_key
+
             summary_imgs = [{**key, **fkey, 'ref_image': ref_image,
```

Comment:
use full words: `summary_images`.  Abbreviations like these do not make the code more readable.